Gruntfile.js

```diff
@@ -1,4 +1,5 @@
-module.exports = function (grunt) {
+module.exports = function (grunt)
```

Comment:
Were we able to successfully connect frontend to bakend using grunt? If not we should remove useless configuration.